src/views/germplasm/BreedingMethods.vue

```diff
@@ -554,7 +554,12 @@ export default class BreedingMethods extends ProgramsBase {
       this.getBreedingMethods();
       this.$emit('show-success-notification', 'Breeding method created successfully');
     } catch (e) {
-      this.$emit('show-error-notification', 'Error while trying to create breeding method');
+      if (e.response.status == 500) {
+        this.$emit('show-error-notification', 'Error while trying to create breeding method');
+      }
+      else{
+        this.$emit('show-error-notification', e.response.data);
+      }
```

Comment:
Why show the error message from the server for all status codes except `500 Internal Server Error`? Unless there's a good reason for this, I would simplify this logic. If it's important to have these separate cases, I think the if/else should be flipped, the message from the server should be shown for specific status codes and the generic error should be a catchall (in the else block).
```suggestion
      this.$emit('show-error-notification', e.response.data);
```